apps/website/src/components/layout/navbar/MobileMenu.tsx

```diff
@@ -1,10 +1,10 @@
-import { Disclosure } from "@headlessui/react";
-import { Fragment } from "react";
-import { signIn, signOut, useSession } from "next-auth/react";
+import { ProfileInfo } from "@/components/layout/navbar/ProfileInfo";
 import { mainNavStructure } from "@/data/navigation";
 import { checkRolesGivePermission, permissions } from "@/data/permissions";
-import { ProfileInfo } from "@/components/layout/navbar/ProfileInfo";
-import { navLinkClassesSub, NavLinkSub } from "./NavLink";
+import { Disclosure } from "@headlessui/react";
+import { signIn, signOut, useSession } from "next-auth/react";
+import { Fragment } from "react";
+import { NavLinkSub, navLinkClassesSub } from "./NavLink";
```

Comment:
Can we sort/group these imports like we do in other files? So packages are in their own group followed by local imports?